src/scss/_variables.scss

```diff
@@ -0,0 +1 @@
+$blue:    #5d95e8;
```

Comment:
Why are you overriding this if you are not using it within your codebase? You need to show you can override as well as using it in your codebase